src/ripple/app/misc/Manifest.h

```diff
@@ -133,7 +133,12 @@ struct ValidatorToken
     std::string manifest;
     SecretKey validationPrivate;
 
-    ValidatorToken(std::string m, SecretKey valPriv);
+private:
+    ValidatorToken(std::string const& m, SecretKey const& valPriv);
```

Comment:
The `SecretKey` member should maybe be names `validationSecret`?  And maybe rename the constructor argument to `valSecret`?